build.gradle

```diff
@@ -1660,6 +1660,8 @@ project(':transaction-coordinator') {
   dependencies {
     implementation libs.jacksonDatabind
     implementation project(':clients')
+    testImplementation libs.junitJupiter
+    testImplementation libs.mockitoCore
     generator project(':generator')
```

Comment:
Please add `testRuntimeOnly runtimeTestLibs` to define the junit platform explicitly